scripts/runfabtests.sh

```diff
@@ -173,7 +175,7 @@ function unit_test {
 	local end_time
 	local test_time
 
-	local e=$(is_excluded $test)
+	local e=$(is_excluded $test_exe)
```

Comment:
No, the exclude format shouldn't change. The `$test` string doesn't have the `fi` prefix prepended. The `$test_exe` does have a bunch of stuff after it, but that's not really different since some of the tests have options appended anyway. By passing `$test_exe` and not `"$test_exe"` I'm only getting the first word in the string (test name).